master/buildbot/data/types.py

```diff
@@ -163,7 +166,9 @@ def __init__(self, len=None, **kwargs):
         self.len = len
 
     def valueFromString(self, arg):
-        val = arg.decode('utf-8')
+        val = arg
```

Comment:
Replacing with ascii2unicode doesn't work here.  Running the tests
I got decoding errors.  I will come up with another solution.